src/cljs/harja/ui/liitteet.cljs

```diff
@@ -99,15 +111,19 @@
   ([tiedosto] (liitetiedosto tiedosto {}))
   ([tiedosto {:keys [salli-poisto? poista-liite-fn nayta-koko?] :as optiot}]
    (let [nimi (:nimi tiedosto)
-         koko (:koko tiedosto)]
+         koko (:koko tiedosto)
+         virustarkastettu? (:virustarkastettu? tiedosto)]
      [:div.liite
       (if (naytettava-liite? tiedosto)
-        [:span
-         [:img.pikkukuva.klikattava {:src (k/pikkukuva-url (:id tiedosto))
-                                     :on-click #(nayta-liite-modalissa tiedosto)}]
-         [:span.liite-nimi (str nimi (when nayta-koko? (str " (" (sievenna-liitteen-koko koko) ") ")))]
-         (when salli-poisto?
-           [liitteen-poisto tiedosto poista-liite-fn])]
+        [:div
+         [:div
```

Comment:
Tarvitaanko kahta diviä päällekkäin?

Reply:
Siinä wrapataan ylemmän divin sisään useampia asioita ja rivityksen takia nuo molemmat tarvitaan.